client/knode/genesis/generator_test.go

```diff
@@ -3,16 +3,19 @@ package genesis
 import (
 	"github.com/stretchr/testify/assert"
 	"testing"
+	"github.com/stretchr/testify/require"
 )
 
 func TestGenerateIsDeterministic(t *testing.T) {
 	options := Networks["kusd"][MainNetwork]
 
 	generatedGenesis, err := Generate(options)
 	assert.NoError(t, err)
+	require.NotNil(t, generatedGenesis)
```

Comment:
Shouldn't the `Generate` return an actual object if there's no error?

Reply:
this would test exactly that, but i guess I can require on the NoError instead

Reply:
yup, that's what I thought. It's fine to have this doublechecked though...